core/src/main/java/org/aerogear/mobile/core/metrics/MetricsService.java

```diff
@@ -62,7 +58,7 @@ public void destroy() {}
      * Send default metrics
      */
     public void sendAppAndDeviceMetrics() {
-        this.publish(defaultMetrics, null);
+        this.publish(INIT_METRICS_TYPE, new Metrics[0], null);
```

Comment:
the `new Metric[0]` would be better to be cached to avoid the continuous creation of new objects.
Add a 
`private static final Metrics[] EMPTY_METRICS = new Metrics[0];`
and pass that to the publish method.

Reply:
+1